tests/test_cookiecutter_xblock.py

```diff
@@ -60,6 +60,8 @@ def fixture_options_baked(cookies_session, request, custom_template):
     baked result.
     """
     with bake_in_temp_dir(cookies_session, extra_context=request.param, template=custom_template):
+        sh.make('upgrade')
+        sh.pip('install', '-r', 'requirements/base.txt')
```

Comment:
Just noticing now that this was base.txt. Does it still work if we switch it to test.txt? :crossed_fingers:

Reply:
Changed. It works.  Now all four copies of this fixture use test.txt.